### DISCUSSION TITLE
Problem with Reac 1

### DISCUSSION THREAD

**Discussion by sudarshan-eshwar** [firemodels/fds#12277] [General]
hey everyone, as a part of my capstone project, __ am trying to simulate a fire in a car. However, I am getting the following error when I run the code: 
ERROR: Problem with REAC 1 (CHID: fire)
ERROR: FDS was improperly set-up - FDS stopped (CHID: fire)
Any help would be greatly appreciated! I have attached a text file of the code that I'm trying to use.
[code.txt](https://github.com/firemodels/fds/files/13623096/code.txt)

**Comment by drjfloyd** [comment]
4.2 Common Error Statements
An FDS calculation may end before the specified time limit. Following is a list of common error statements
and how to diagnose the problems:
Input File Errors: The most common errors in FDS are due to mis-typed input statements. These errors
result in the immediate halting of the program and a statement like, “ERROR: Problem with the HEAD
line.” For these errors, check the line in the input file named in the error statement. Make sure the
parameter names are spelled correctly. Make sure that a / (forward slash) is put at the end of each
namelist entry. Make sure that the right type of information is being provided for each parameter, like
whether one real number is expected, or several integers, or whatever. Make sure there are no non-ASCII
characters being used, as can sometimes happen when text is cut and pasted from other applications or
word-processing software. Make sure zeros are zeros and O’s are O’s. Make sure 1’s are not !’s. **Make
sure apostrophes are used to designate character strings.** Make sure the text file on a Unix/Linux machine
was not created on aWindows machine, and vice versa. Make sure that all the parameters listed are still
being used – new versions of FDS often drop or change parameters forcing you to re-examine old input
files.

&REAC FUEL=**'PROPANEâ€™,** SOOT_YIELD = 0.062, CO_YIELD = 0.0705, RADIATIVE_FRACTION=0.35/
